### ISSUE TITLE
Can't turn off Full Screen

### ISSUE THREAD

**Issue by Cetins** [LunaPixelStudios/Better-Minecraft#165]
### Describe the Bug

When I turn off the full screen. only the bottom task bar becomes visible on the regular full screen mod. You can see on the image the buttons are slightly behind the taskbar. When i take a screenshot of my laptop screen it doesn't show a difference as the screenshots don't save the taskbar. Also I can see the window properties such as minimize, close the window etc. And the hot bar also stays behind the taskbar when Full screen turned off.

### Reproduction Steps

1.closed the game and curseforge loader
2.restart the laptop
3.open the curseforge launcher and start the game through
...

### Screenshots and Videos

![20220227_134305](https://user-images.githubusercontent.com/69481095/155885186-24238a06-45e1-43aa-ba93-33a90518dd07.jpg)
![20220227_135016](https://user-images.githubusercontent.com/69481095/155885190-2a309383-2c8f-4532-b42d-fb63506b0330.jpg)

### Operating System

Windows 11

### Minecraft Version

1.18

### Modloader

Fabric

### Modpack Version

V17

### Optifine

I used Optifine

### Other Mods

just better minecraft fabric nothing else but i''ve deleted double doors mod.

**Comment by SharkieTV** [comment]
Press F11

**Comment by Cetins** [comment]
This is the F11 version. I've turn off/on the fullscreen from menu and thereafter I've spammed F11. Still can't see the window options (close/minimize/maximize). The options in the menu and the F11 only enables the computers taskbar and the meantime game hotbar still stays behind that taskbar. But I've installed older versions of your modpack because it wasn't like that before. Then I see that older versions too have the same problem now. I don't think it's your modpack, I think it has to be something with windows causing this.
